### PR TITLE
feat(core): Add API to request an anchor

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Comment by stbrody** [comment]
CC @oed just a heads up that this adds a new HTTP endpoint

File: packages/common/src/stream.ts [hunk 1]
```diff
@@ -224,6 +224,10 @@ export abstract class Stream extends Observable<StreamState> implements StreamSt
     this.state$.next(stream.state)
   }
 
+  async requestAnchor(): Promise<AnchorStatus> {
+    return this.api.requestAnchor(this.id)
+  }
```
Comment (oed) [inline packages/common/src/stream.ts:229]: Should we add the same thing to caip10-link?

**Review by oed — COMMENTED** [review]
Thanks, is was needed.

File: packages/common/src/stream.ts (hunk 1 above)
Comment (stbrody) [reply packages/common/src/stream.ts:229]: This is in the Stream base class, so it's available to all StreamType implementations

Comment (oed) [reply packages/common/src/stream.ts:229]: Ah, my bad!

**Review by ukstv — APPROVED** [review]

**Pull request merged**